.yamato/mobile-build-and-test.yml

```diff
@@ -0,0 +1,90 @@
+2021.2_Build_Android_player:
+  name: 2021.2 Build Android player
+  agent:
+      type: Unity::VM
+      # Any generic image can be used, no need to have Android tools in the image for building
+      # All Android tools will be downloaded by unity-downloader-cli
+      image: package-ci/win10:stable
+      flavor: b1.small
+
+  commands:
+    - pip install unity-downloader-cli --index-url https://artifactory.prd.it.unity3d.com/artifactory/api/pypi/pypi/simple
+    - mkdir C:\TMP
+    - cd C:\TMP && unity-downloader-cli -u 2021.2 -c editor -c Android -w --fast
+    - curl -s https://artifactory.prd.it.unity3d.com/artifactory/unity-tools-local/utr-standalone/utr.bat --output utr.bat
+    - utr.bat --suite=playmode --platform=Android --editor-location=C:\TMP\.Editor --testproject=testproject --player-save-path=build/players --artifacts_path=build/logs --scripting-backend=mono --build-only --testfilter=Unity.Netcode.RuntimeTests
+  artifacts:
+    players:
+        paths:
+          - "build/players/**"
+    logs:
+        paths:
+          - "build/logs/**"
+
+2021.2_Build_iOS_player:
+  name: 2021.2 Build iOS player
+  agent:
+    type: Unity::VM::osx
+    image: mobile/macos-10.15-testing:stable
+    flavor: b1.large
+  commands:
+    - pip install unity-downloader-cli --index-url https://artifactory.prd.it.unity3d.com/artifactory/api/pypi/pypi/simple
+    - unity-downloader-cli -u 2021.2 -c editor -c iOS -w --fast
+    - curl -s https://artifactory.prd.it.unity3d.com/artifactory/unity-tools-local/utr-standalone/utr --output utr
+    - chmod +x ./utr
+    - ./utr --suite=playmode --platform=iOS --editor-location=.Editor --testproject=testproject --player-save-path=build/players --artifacts_path=build/logs --build-only --testfilter=Unity.Netcode.RuntimeTests
+  artifacts:
+    players:
+        paths:
+          - "build/players/**"
+    logs:
+        paths:
+          - "build/logs/**"
+
+
+2021.2_Run_iOS_Player_With_Tests:
+  name: 2021.2 Run iOS player with tests
+  agent:
+      type: Unity::mobile::iPhone
+      model: SE
+      image: mobile/macos-10.15-testing:stable
+      flavor: b1.medium
+  # Set a dependency on the build job
+  dependencies:
+    - .yamato/mobile-build-and-test.yml#2021.2_Build_iOS_player
+  commands:
+    # Download standalone UnityTestRunner
+    - curl -s https://artifactory.prd.it.unity3d.com/artifactory/unity-tools-local/utr-standalone/utr --output utr
```

Comment:
Same as above technically you could get a different version of UTR between build and run which I assume you do not want so I would here again recommend sticking to a specific version of UTR. Here is the script we use to download it for UTP https://github.cds.internal.unity3d.com/unity/com.unity.transport/blob/master/Tools~/CI/download_utr.py